en/components/autocomplete.md

```diff
@@ -0,0 +1,262 @@
+        [igxAutocomplete]='townsPanel'
+        [igxAutocompleteDisabled]="disabled"/>
+    <label igxLabel for="towns">Towns</label>
+</igx-input-group>
+<igx-drop-down #townsPanel>
+    <igx-drop-down-item *ngFor="let town of towns">
+        {{town}}
+    </igx-drop-down-item>
+</igx-drop-down>
+<igx-switch name="toggle" [(ngModel)]="!disabled"></igx-switch>
+```
+
+```typescript
+@Component({
+    selector: 'app-autocomplete-sample',
+    styleUrls: ['autocomplete.sample.css'],
+    templateUrl: `autocomplete.sample.html`
+})
+export class AutocompleteSampleComponent {
+    disabled;
+    constructor() {
+        this.towns = [ 'Sofia', 'Plovdiv', 'Varna', 'Burgas', "Ruse", "Stara Zagora",
+            "Pleven", "Dobrich", "Sliven", "Shumen", "Pernik", "Haskovo", "Yambol"];
+    }
+}
+```
+
+### Drop-down settings
+The `igx-autocomplete` drop-down positioning, scrolling strategy and outlet can be configured using, the [`IgxAutocompleteSettings`]({environment:angularApiUrl}/classes/igxautocompletedirective.html#autocompleteSettings) option. It allows values from type [`AutocompleteOverlaySettings`]({environment:angularApiUrl}/classes/autocompleteoverlaysettings.html).
+
+The following example displays that the positioning of the drop-down can be set to be always above the input, where the directive is applied. It also disables opening and closing animations. For that purpose the `ConnectedPositioningStrategy` is used:
+
+```html
+<igx-input-group #inputGroup>
+    <input igxInput name="towns" type="text"
+        [igxAutocomplete]='townsPanel'
+        [igxAutocompleteSettings]='settings'/>
+    <label igxLabel for="towns">Towns</label>
+</igx-input-group>
+<igx-drop-down #townsPanel>
+    <igx-drop-down-item *ngFor="let town of towns">
+        {{town}}
+    </igx-drop-down-item>
+</igx-drop-down>
+```
+
+```typescript
+@Component({
+    selector: 'app-autocomplete-sample',
+    styleUrls: ['autocomplete.sample.css'],
+    templateUrl: `autocomplete.sample.html`
+})
+export class AutocompleteSampleComponent {
+    constructor() {
+        this.towns = [ 'Sofia', 'Plovdiv', 'Varna', 'Burgas', "Ruse", "Stara Zagora",
+            "Pleven", "Dobrich", "Sliven", "Shumen", "Pernik", "Haskovo", "Yambol"];
+    }
+    @ViewChild('inputGroup', { read: IgxInputGroupComponent }) inputGroup: IgxInputGroupComponent;
+
+    this.settings = {
+        positionStrategy: new ConnectedPositioningStrategy({
+            closeAnimation: null,
+            openAnimation: null,
+            verticalDirection: VerticalAlignment.Top,
+            verticalStartPoint: VerticalAlignment.Top
+        })
+    };
+}
+```
+
+> Note: The default positioning strategy is `AutoPositionStrategy` and the drop-down is opened according to the available space.
+
+## Application Scenarios
+### Grouped igxAutocomplete, inside Reactive From
+
+The following sample demonstrates an autocomplete that allows selecting a cinema, where movies are grouped by a city:
```

Comment:
Let's include guidance/example what to be typed - because the users might not know the Cinema name in that specific case.